docs/docs-content/clusters/data-center/vmware/create-manage-vmware-clusters.md

```diff
@@ -0,0 +1,218 @@
+
+   Click on the **Next** button when you are done.
+
+10. You can configure the following cluster management features now if needed, or you can do it later:
+
+    - OS Patching
+    - Schedule scans
+    - Schedule backups
+    - Role Based Access Control (RBAC)
+    - Location
+
+    #### OS Patching
+
+    Specify your preferred **OS Patching Schedule** for the cluster. Check out the
+    [OS Patching](../../cluster-management/os-patching.md) page to learn more about OS patching.
+
+    #### Scan Options
+
+    Enable any scan options you want Palette to perform, and select a scan schedule. Palette provides support for
+    Kubernetes configuration security, penetration testing, and conformance testing.
+
+    #### Backup Options
+
+    Schedule any backups you want Palette to perform. Review
+    [Backup and Restore](../../cluster-management/backup-restore/backup-restore.md) for more information.
+
+    #### RBAC Configuration
+
+    RBAC configuration is required when you configure custom OIDC. You must map a set of users or groups to a Kubernetes
+    RBAC role. To learn how to map a Kubernetes role to users and groups, refer to
+    [Create Role Bindings](../../cluster-management/cluster-rbac.md#create-role-bindings). Refer to
+    [Use RBAC with OIDC](../../../integrations/kubernetes.md#use-rbac-with-oidc) for an example.
+
+    #### Location
+
+    Specify the location of the cluster. The cluster location is added to the project dashboard location map.
+
+11. Click on the **Validate** button and review the cluster configuration and settings summary.
+
+12. Click **Finish Configuration** to deploy the cluster.
+
+The cluster deployment process is initiated. You can monitor the cluster deployment progress by navigating to the left
+**Main Menu** and selecting **Clusters**. Click on the cluster you just created to view the cluster details page. The
+**Cluster Status** field displays the current status of the cluster.
+
+## Validate
+
+Use the following steps to validate that the cluster is available and healthy.
+
+1. Log in to [Palette](https://console.spectrocloud.com).
+
+2. Navigate to the left **Main Menu** and click **Clusters**.
+
+3. The **Clusters** page lists all available clusters that Palette manages. Select the cluster you deployed to review
+   its details page.
+
+4. Ensure the **Cluster Status** field contains the value **Running**.
+
+:::tip
+
+You can download the cluster's kubeconfig file to access the cluster using the Kubernetes command-line tool, kubectl.
+Check out the [Access a Cluster](../../cluster-management/palette-webctl.md) guide to learn how to download the
+kubeconfig file.
+
+:::
+
+## Next Steps
+
+Now that you have a Kubernetes cluster deployed, you can start developing and deploying applications to your clusters.
+We recommend you review the day two responsibilities and become familiar with the cluster management tasks. Check out
```

Comment:
```suggestion
We recommend you review the Day-2 responsibilities and become familiar with the cluster management tasks. Check out
```

Reply:
This is how this is referred to on this page https://docs.spectrocloud.com/clusters/cluster-management/

Reply:
Excellent catch